src/averageAges.js

```diff
@@ -20,28 +20,64 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const menLength = people.filter(person => person.sex === 'm').length;
+
+  const menCenturyLength = people.filter(person =>
+    person.sex === 'm' && Math.ceil(person.died / 100) === century).length;
```

Comment:
```suggestion
    person.sex === 'm' && Math.ceil(person.died / 100) === century)
    .length;
```
Методы, также как атрибуты html, переносим на новую строку. А то он где-то там далеко затесался и сразу непонятно что этот код делает